tests/profiling/collector/test_memalloc.py

```diff
@@ -163,11 +161,11 @@ def test_memory_collector():
         last_call = event.frames[0]
         assert event.size > 0
         if last_call[2] == "<listcomp>" and last_call[1] == _ALLOC_LINE_NUMBER:
-            assert event.thread_id == nogevent.main_thread_id
+            assert event.thread_id == threading.main_thread().ident
             assert event.thread_name == "MainThread"
             count_object += 1
             assert event.frames[2][0] == __file__
-            assert event.frames[2][1] == 154
+            assert event.frames[2][1] == 152
             assert event.frames[2][2] == "test_memory_collector"
 
     assert count_object > 0
```

Comment:
Should we also remove the `@pytest.mark.skipif(TESTING_GEVENT...)` for the subsequent test `test_memory_collector_ignore_profiler`?